.github/workflows/scripts/issues/orderIdValidation.js

```diff
@@ -12,7 +12,11 @@ module.exports = async ({ core, context, github }) => {
     const issueNumber = context.issue.number;
 
     const orderId = process.env.ORDER_ID;
-    const orderApiToken = process.env.ORDER_API_TOKEN;
+
+    const orderApiToken = await github.rest.actions.getOrgSecret({
+      owner,
+      secret_name: 'SHOP_AUTH_TOKEN',
+    });
```

Comment:
Will this work? 🤔

Reply:
I honestly don't know exactly ... I'll give it a shot and otherwise will turn back to the env variable! :P

Reply:
its documented very poorly how those permissions work .. especially since you have several different flavors of tokens

Reply:
yeah, passing env between them is annoying. IIRC is the env is in the caller workflow, eg: mui-x, it should be present when it calls this. Else you might need to add it as inputs/secrets eg:

```yaml
// mui-x

name: New issue triage
on:
  issues:
    types:
      - opened

permissions: {}

jobs:
  issue_cleanup:
    name: Clean issue body
    uses: mui/mui-public/.github/workflows/issues_body-cleanup.yml@master
    permissions:
      contents: read
      issues: write
  order_id_validation:
    name: Validate order ID
    needs: issue_cleanup
    if: needs.issue_cleanup.outputs.orderId != ''
    uses: mui/mui-public/.github/workflows/issues_order-id-validation.yml@master
    with:
      orderId: ${{ needs.issue_cleanup.outputs.orderId }}
      apiKey: ${{ env.API_KEY }}
    permissions:
      contents: read
      issues: write
    secrets:
      apiKey: ${{ secrets.API_KEY }}

```